configure

```diff
@@ -990,4 +993,11 @@ else:
 
 gyp_args += args
 
+if warned:
+  if os.isatty(1):
+    msg = '\033[1m\033[93mCareful! Warnings happened during \'configure\'.\033[0m'
+  else:
+    msg = 'Careful! Warnings happened during \'configure\'.'
+  print msg
```

Comment:
Redundancy isn't bad per se.  Maybe `warn('warnings were emitted in the configure step')`?